### PR TITLE
fix(notification): add correct icon svgs for different types

### LINKED ISSUE #1327: Notifications are showing the same icon
screen shot 2018-10-24 at 11 22 47 am

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #1327 

Adds missing icons for different notification types
screen shot 2018-10-24 at 11 36 13 am

### CODE REVIEW

File: src/components/notification/notification.hbs [hunk 1]
```diff
@@ -2,9 +2,26 @@
...
       {{#is ../variant "inline"}}
-        <svg class="bx--{{../variant}}-notification__icon" width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
-          <path d="M8 16A8 8 0 1 1 8 0a8 8 0 0 1 0 16zM3.293 4.707l8 8 1.414-1.414-8-8-1.414 1.414z" fill-rule="evenodd" />
-        </svg>
+        {{#is type "error"}}
+          <svg class="bx--{{../variant}}-notification__icon" width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+            <path d="M8 16A8 8 0 1 1 8 0a8 8 0 0 1 0 16zM3.293 4.707l8 8 1.414-1.414-8-8-1.414 1.414z" fill-rule="evenodd" />
+          </svg>
+        {{/is}}
+         {{#is type "info"}}
```
Comment (asudoh) [inline src/components/notification/notification.hbs]:
```suggestion
        {{#is type "info"}}
```

File: src/components/notification/notification.hbs [hunk 2]
```diff
@@ -2,9 +2,26 @@
...
+          <svg class="bx--{{../variant}}-notification__icon" width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+            <path d="M8 16A8 8 0 1 1 8 0a8 8 0 0 1 0 16zm1-3V7H7v6h2zM8 5a1 1 0 1 0 0-2 1 1 0 0 0 0 2z"></path>
+          </svg>
+        {{/is}}
+         {{#is type "success"}}
```
Comment (asudoh) [inline src/components/notification/notification.hbs]:
```suggestion
        {{#is type "success"}}
```

File: src/components/notification/notification.hbs [hunk 3]
```diff
@@ -2,9 +2,26 @@
...
+          <svg class="bx--{{../variant}}-notification__icon" width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+            <path d="M8 16A8 8 0 1 1 8 0a8 8 0 0 1 0 16zm3.293-11.332L6.75 9.21 4.707 7.168 3.293 8.582 6.75 12.04l5.957-5.957-1.414-1.414z"></path>
+          </svg>
+        {{/is}}
+         {{#is type "warning"}}
```
Comment (asudoh) [inline src/components/notification/notification.hbs]:
```suggestion
        {{#is type "warning"}}
```

**Review by asudoh — APPROVED** [review]
LGTM except suggested indents - Thanks @alisonjoseph for fixing and my apologies for my oversight during my Handlebars conversion effort!

**Review by joshblack — APPROVED** [review]

**Pull request merged**